Support for v2 Timer Creation: client method + payload helpers (#887)
globus/globus-sdk-python: docs/services/timer.rst

Helper Objects
--------------

> The main helper users should use is the one for constructing Transfer Timers:

Reviewer: ```suggestion
A helper is provided for constructing Transfer Timers:
```

.. autoclass:: TransferTimer
   :members:
   :show-inheritance:

In order to schedule a timer, pass a ``schedule`` with relevant parameters.
This can be done using the two schedule helper classes

.. autoclass:: OnceTimerSchedule
   :members:
   :show-inheritance:

.. autoclass:: RecurringTimerSchedule
   :members:
   :show-inheritance:

TimerJob (legacy)
~~~~~~~~~~~~~~~~~

> The ``TimerJob`` class is still supported for creating timer, but it is

Reviewer: ```suggestion
The ``TimerJob`` class is still supported for creating timers, but it is
```
